ArduSub/failsafe.cpp

```diff
@@ -321,7 +321,8 @@ void Sub::failsafe_gcs_check()
     uint32_t tnow = AP_HAL::millis();
 
     // Check if we have gotten a GCS heartbeat recently (GCS sysid must match SYSID_MYGCS parameter)
-    if (tnow - gcs_last_seen_ms < FS_GCS_TIMEOUT_MS) {
+    const float validated_timeout = constrain_float(g.failsafe_gcs_timeout, 2.0, 120);
```

Comment:
I really have my reservations about these constraints - particularly the lower one.

This is magic stuff happening in the code, there's nothing the user can know about this from the parameter description.

I don't think it's unreasonable for the user to set outside the bounds you've put in here.

Are you sure you want to bound things here?

Reply:
I agree with you.
I though I had seen that somewhere else but I cant find it anywhere...
Anyway. code updated, thanks for the review